common.sh

```diff
@@ -149,8 +149,8 @@ function build_kb {
   if [ "$INJECT_KB_VERSION" = "unknown" ]; then
     opts=""
   else
-    # TODO: what does this thing do.
-    opts=-ldflags "-X sigs.k8s.io/kubebuilder/v2/cmd/version.kubeBuilderVersion=$INJECT_KB_VERSION"
+    # Injects the version into the cmd/version.go file
+    opts=-ldflags "-X sigs.k8s.io/kubebuilder/v2/cmd.kubeBuilderVersion=$INJECT_KB_VERSION"
```

Comment:
Wdyt about adding adding something like this to `make build`, ex.
```make
BUILD_VERSION=$(shell git descripe --dirty --tags --always)
BUILD_COMMIT=$(shell git rev-parse HEAD)
LD_FLAGS=-ldflags "-X sigs.k8s.io/kubebuilder/v2/cmd.kubeBuilderVersion=$(BUILD_VERSION) -X sigs.k8s.io/kubebuilder/v2/cmd.gitCommit=$(BUILD_COMMIT)"
build: ## Build the project locally
        go build $(LD_FLAGS) -o bin/kubebuilder ./cmd
```
perhaps in a follow-up?

Reply:
I tracked this change in an Issue some time ago: #1401